net-misc/curl/curl-7.64.1.recipe

```diff
@@ -84,7 +84,7 @@ BUILD()
 	certsDir="$certsInstallDir/$relativeDataRootDir/ssl"
 
 	autoconf
-	CFLAGS=-D_BSD_SOURCE CXXFLAGS=-D_BSD_SOURCE runConfigure ./configure \
+	CPPFLAGS=-D_BSD_SOURCE runConfigure ./configure \
```

Comment:
Why?

Reply:
CPPFLAGS is now defined for -D, -I placement by compiler (CFLAGS generates warnings otherwise).

Reply:
Compare the generated configure.h

Reply:
can we merge? CPPFLAGS affects the preprocessing, so this is OK.